behaviours/meme_rater.py

```diff
@@ -276,9 +277,22 @@ async def delete_meme(event: hikari.GuildReactionAddEvent) -> None:
     if not shit_reaction:
         return
 
-    count = shit_reaction.count
+    # Find the "🔟" reaction; if not found then return 0.
+    ten_reaction = next(
+        (reaction for reaction in message.reactions if reaction.emoji == "🔟"), 0
+    )
+
+    # Convert to int var
+    ten_reaction_count = (
+        ten_reaction.count if isinstance(ten_reaction, hikari.messages.Reaction) else 0
+    )
+
+    count = (
+        shit_reaction.count - ten_reaction_count
+    )  # Final count of shit emojis offset by 10s.
     if not shit_reaction.is_me:
         count += 1
+
     if count >= int(os.getenv("MEME_VOTE_DELETE_THRESHOLD", 4)):
```

Comment:
also rename count to something else if 2 things getting counted now (or net_poo_count)